content/webapp/components/IIIFItemList/IIIFItemList.tsx

```diff
@@ -34,6 +34,7 @@ const IIIFItemList: FunctionComponent<Props> = ({
                   item={item}
                   canvas={canvas}
                   i={i}
+                  titleOverride={`${i + 1}/${canvases?.length}`}
```

Comment:
Would you have an example of where this the one that comes through?

Reply:
/works/b4tj49m4/items

Reply:
Thank you, yes it's much nicer this way!